File: Cloud5S_WEB/src/app/@filter/Business/contract.filter.ts
import {BaseFilter} from '../Common/base.filter';
import * as moment from 'moment';

export class ContractFilter extends BaseFilter {
  selectedRange: Date[] = [moment().subtract(3, 'days').startOf('day').toDate(), moment().endOf('day').toDate()];
  code?: string = '';
  Type: string = '';
  PartnerCode: string = '';
}

export class ContractFilterRequest extends BaseFilter {
  FromDate?: string = moment().subtract(3, 'days').format('YYYY-MM-DD');
  ToDate?: string = moment().format('YYYY-MM-DD');
  Type: string = '';
  PartnerCode: string = '';
}

File: Cloud5S_API/DMS.Business/Filter/BU/ContractFilter.cs
using DMS.BUSINESS.Filter.Common;

namespace DMS.BUSINESS.Filter.BU
{
    public class ContractFilter : BaseFilter
    {
        public string Type { get; set; }
        public DateTime? FromDate { get; set; }
        public DateTime? ToDate { get; set;}
        public string PartnerCode { get; set; }
    }

    public class ContractExportFilter
    {
        public string Type { get; set; }
        public DateTime? FromDate { get; set; }
        public DateTime? ToDate { get; set; }
        public string PartnerCode { get; set; }

        public string KeyWord { get; set; }
    }
}
